ext-apiserver/pkg/apis/cluster/v1alpha1/cluster_types_test.go

```diff
@@ -15,65 +14,68 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-
 package v1alpha1_test
 
 import (
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	. "k8s.io/kube-deploy/ext-apiserver/pkg/apis/cluster/v1alpha1"
```

Comment:
If you change the package statement from `package v1alpha1_test` to `package v1alpha1`, you don't have to do this import. Using the `package pkgname_test` pattern is usually only used to enforce black-box testing, or more commonly, to avoid import cycles.